app/controllers/scans_controller.rb

```diff
@@ -33,17 +33,30 @@ def current_request
 
   protected
 
+  def illiad_page
+    @illiad_page = { 'Action': '10', 'Form': '30', 'rft.genre': 'scananddeliverArticle' }
```

Comment:
In the context of the controller you typically set an instance variable (`@variable`) to make something accessible to the view layer.  Since you're calling these method directly you can simply use the return value of the method and don't need to set any instance variable.